src/main/java/core/basesyntax/model/io/FileWriterImpl.java

```
@@ -0,0 +1,15 @@
+package core.basesyntax.model.io;
+
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String filePath) {
+        try (BufferedWriter writer = new BufferedWriter(new java.io.FileWriter(filePath))) {
```

Good: try-with-resources is used and IOException is rethrown as a RuntimeException with an informative message. This follows the checklist guidance about using try-with-resources and not ignoring exceptions. Keep this behavior.